setup.php

```diff
@@ -1574,14 +1570,14 @@ function cloneClicked() {
     <script>
         //jquery-ui tooltip
         $(function() {
-            $('.icon-tooltip').prop( "title", "Click to see more information").tooltip({
+            $('.icon-tooltip').attr({"title": <?php echo xlj('Click to see more information'); ?>, "data-toggle":"tooltip", "data-placement":"bottom"}).tooltip({
```

Comment:
Note that the translation functions are not available in this script since there is no database yet. Would revert the xlj change here.